java/bed-allocation/README.adoc

```diff
@@ -50,9 +50,60 @@ Then try _live coding_:
 Notice that those changes are immediately in effect.
 
 [[enterprise]]
-== Run the application with the Enterprise Edition
+== Run the application with Timefold Solver Enterprise Edition
 
-For high-scalability use cases, try the paid https://docs.timefold.ai/timefold-solver/latest/enterprise-edition/enterprise-edition[Enterprise Edition] instead:
+For high-scalability use cases, switch to https://docs.timefold.ai/timefold-solver/latest/enterprise-edition/enterprise-edition[Timefold Solver Enterprise Edition],
+our commercial offering.
+https://timefold.ai/contact[Contact Timefold] to obtain the credentials required to access our private Enterprise Maven repository.
+
+. Create `.m2/settings.xml` in your home directory with the following content:
++
+[tabs]
+====
+Maven::
```

Comment:
Do tabs work on Github?
Arguably if the quickstart doesn't have support for Gradle, this can be simplified by not including Gradle instructions.

Reply:
It makes sense. I'll remove it from the modules that do not use Gradle. Currently, there is no support for tab in the GH. Here is a visual representation of what it looks like:
![image](https://github.com/TimefoldAI/timefold-quickstarts/assets/11961659/169c78eb-ed90-4008-bd26-778b66d9e0e0)